<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UART_ConnectionWindow</class>
 <widget class="QDialog" name="UART_ConnectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UART_ConnectionWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="aircoolcontroll.qrc">
    <normaloff>:/Images/connect.png</normaloff>:/Images/connect.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Port</string>
           </property>
           <property name="buddy">
            <cstring>comboBoxCOM</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxCOM">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Transmition Speed</string>
           </property>
           <property name="buddy">
            <cstring>comboBoxSpeed</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxSpeed"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Device ID</string>
           </property>
           <property name="buddy">
            <cstring>spinBoxID</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxID">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="UARTconnectButton">
         <property name="text">
          <string>Connect to Device</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="communicationErrorLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="aircoolcontroll.qrc"/>
 </resources>
 <connections/>
</ui>
